test/quantization/fx/test_quantize_fx.py

```diff
@@ -2542,6 +2542,156 @@ def forward(self, x):
             mp(torch.rand(4, 4, 4, 4))
             mc = convert_fx(mp)
 
+    def _check_not_observed(self, model: nn.Module, node_name_list: List[str]):
+        # check that no obsersvers have the named nodes as an input
+        for node in model.graph.nodes:
+            if hasattr(model, node.name) and isinstance(
```

Comment:
looks like we are looking for observers here, I think a better way to do it would be to check:

```
named_modules = model.named_modules()
if node.op == "call_module" and is_activation_post_process(named_modules[node.target]):
    ...
```
also in the checking we can rely on the structure of the graph, instead of node name, e.g. we can check if the value that is observed is the 3rd argument of some op etc.

Reply:
can you clarify what you mean by rely on structure of the graph? Are you saying I should search for the op in question (i.e. view) and then go through the non_tensor args and see if the source of any of those are an activation_post_process?